downstream/modules/platform/ref-access-control-requirements.adoc

```diff
@@ -0,0 +1,180 @@
+
+[id="ref-access-control-requirements_{context}"]
+
+= Access control requirements
+
+[role="_abstract"]
+
+{PlatformName} uses a number of ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by a firewall.
```

Comment:
```suggestion
{PlatformName} uses a number of ports to communicate with its services. Ensure that these ports are open and available in the firewall settings to allow for incoming connection to the {PlatformName} server.
```

Reply:
Just a suggestion. There may be a better way to phrase this, but I wanted to try to consolidate the sentences.